controller/handler_edge_ctrl/common_tunnel.go

```diff
@@ -150,6 +150,29 @@ func (self *baseTunnelRequestContext) ensureApiSession(configTypes []string) boo
 	return false
 }
 
+func (self *baseTunnelRequestContext) loadServiceForName(name string) {
+	if self.err == nil {
+		var err error
+		self.service, err = self.handler.getAppEnv().Handlers.EdgeService.ReadByName(name)
+
+		if err != nil {
+			if boltz.IsErrNotFoundErr(err) {
+				err = InvalidServiceError{}
+			} else {
+				err = internalError(err)
+			}
+
+			logrus.
```

Comment:
We were logging self.session.Id and self.session.ServiceId and self.session wasn't set yet at this point.